modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -104,6 +110,9 @@ public class ClientExtensionProjectConfigurator
 	public static final String CREATE_CLIENT_EXTENSION_CONFIG_TASK_NAME =
 		"createClientExtensionConfig";
 
+	public static final String GENERATE_LANGUAGE_BATCH_TASK_NAME =
+		"generateLanguageBatch";
```

Comment:
This task name should match the task class name. We should go with whichever is the most accurate.